nmostesting/suites/BCP0060102Test.py

```diff
@@ -913,3 +913,134 @@ def test_04(self, test):
 
         except TestingFacadeException as e:
             return test.UNCLEAR(e.args[0])
+
+    def test_05(self, test):
+        """
+        Instruct Receiver to subscribe to a Sender's JPEG XS Flow via IS-05
+        """
+        # Perform an immediate activation between a Receiver and a Sender.
+        # Sender and Recievers have SDP/caps as specifiied in TR-08.
+
+        CANDIDATE_SENDER_COUNT = 3
```

Comment:
Why 3?

Reply:
Because [3 Is The Magic Number](https://www.youtube.com/watch?v=YZoYEr6NdmE)...well *a* magic number at least.  I will replace this with a 'curated' list of representative operability points.

Reply:
Done.